### DISCUSSION TITLE
Cannot see videos in the download folder

### DISCUSSION THREAD

**Discussion by vatsalrp** [shaked6540/YoutubePlaylistDownloader#148] [Q&A]
Downloaded in default directory and then again in another folder but cannot see any files in those folders, also the space is occupied. Help!

**Comment by vatsalrp** [comment]
> Downloaded in default directory and then again in another folder but cannot see any files in those folders, also the space is occupied. Help!

**Comment by shaked6540** [comment]
See https://github.com/shaked6540/YoutubePlaylistDownloader/issues/93 .
In short, run the program as administrator or save in a different drive
